### PR TITLE
docs: adding HIL sanity badges 

### PR DESCRIPTION
**Pull request by milapjoshi**
## Summary

- one for last stable and one for current state 
- removing PR welcome badge as discussed with brian/tariq

We are running nightly HIL testing. so using these badge, we want to achive two things. 
Indicate last HIL sanity stable build, which can be used to create quick release branch.
Indicate latest hil testing status.

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by hcgatewood — CHANGES_REQUESTED** [review]
This feels like a weird addition, or at least one that's eng-driven rather than user-driven. The top of a readme is meant to communicate information to top-of-funnel

1. The badge text doesn't convey semantically meaningful information to new users -- what does "AGW HIL sanity" mean to a common user? Is this the latest "commit that's known stable via basic HIL testing", or ?
2. The badge link takes you to a (FB-internal ?) Grafana login page, and I have no idea how to log in (I'm logged in to my personal grafana, but that seems ineffective)
3. If you want to remove the "PRs welcome" badge, please provide explicit rationale

For those reasons, I think this type of badge is a better fit for the Docusaurus, which targets middle or bottom of funnel, rather than the root readme, which targets top of funnel. You can also place context and directives for how to use the badge if we go with the Docusaurus approach

**Comment by milapjoshi** [comment]
> This feels like a weird addition, or at least one that's eng-driven rather than user-driven. The top of a readme is meant to communicate information to top-of-funnel
> 
> 1. The badge text doesn't convey semantically meaningful information to new users -- what does "AGW HIL sanity" mean to a common user? Is this the latest "commit that's known stable via basic HIL testing", or ?
mj>>  "Is this the latest "commit that's known stable via basic HIL testing", or ? " >> yes, we can rephrase or provide link in docusaurus to provide more description. basically, we want to make community aware about 'Hardware in loop' testing done on magma main line build.
> 2. The badge link takes you to a (FB-internal ?) Grafana login page, and I have no idea how to log in (I'm logged in to my personal grafana, but that seems ineffective)
mj>> badge link is open to public. we can provide credential for any way to access it. like circleci access control. (As of now its static shared credential)
> 3. If you want to remove the "PRs welcome" badge, please provide explicit rationale
MJ>> This came from brian. Rational is -  badge on root READme file is prime real estate. So, if PR welcome is not changing to any other message during merge freeze it can be removed to make space.
> 
> For those reasons, I think this type of badge is a better fit for the Docusaurus, which targets middle or bottom of funnel, rather than the root readme, which targets top of funnel. You can also place context and directives for how to use the badge if we go with the Docusaurus approach

MJ>> ok let me take this action items :
   - work with Jerad to find proper place in docusaurus to provide more information about HW in loop testing.
   - one badge of latest HIL sanity run (pass/fail) can be still on root readme with link to docusaurus 
   - one badge about last HIL sanity passed build would be place on internal docusaurus page.

**Review by hcgatewood — COMMENTED** [review]
Can you give screenshots of what these changes look like 🙂

**Comment by milapjoshi** [comment]
@hcgatewood  you think this approach is ok. i have readme file with details of what HIL is and badge is moved inside that readme file. however, we still have top of the funnel badge to indicate hil sanity status. with link to internal readme page.

**Comment by milapjoshi** [comment]
> Can you give screenshots of what these changes look like 🙂

image

image

https://github.com/magma/magma/blob/hil_badge/docs/readmes/lte/hil_tests.md

**Comment by milapjoshi** [comment]
@hcgatewood can you please check on this.

**Review by hcgatewood — CHANGES_REQUESTED** [review]
This looks reasonable. Couple notes

- Spelling/grammar
- Dashboard still isn't accessible by me directly, and there are no instructions on how to access it
- The word "sanity" can be removed everywhere -- just call it "HIL AGW tests" or similar

**Comment by milapjoshi** [comment]
> This looks reasonable. Couple notes
> 
> * Spelling/grammar
> * Dashboard still isn't accessible by me directly, and there are no instructions on how to access it
> * The word "sanity" can be removed everywhere -- just call it "HIL AGW tests" or similar
@hcgatewood 
--removed sanity and corrected some typo. (Badges would update with nightly run, and would come as `HIL AGW tests`
dashboard should be accessible. i added some credential details. its view only cred.
you can check here - https://github.com/magma/magma/tree/hil_badge

File: docs/readmes/lte/hil_tests.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+1. Verify 600 UEs at 5 UE/sec can connect to a Magma Access Gateway
+1. Verify 200 UEs across 12 eNodeBs with 2M data per UE
+1. Verify 400 UEs across 12 eNodeBs with 500k data per UE
+1. Verify 600 UEs across 12 eNodeBs with 500k data per UE
+1. Verify 30 UEs across 12 eNodeBs with 500K data changing state from active-idle-active-idle
+
+### Dashboard
+
+All test results are available to compare on [dashboard](http://automation.fbmagma.ninja/). Please use `username:magma` and `password:magma`.
+We can retrieve log and Grafana metrics for each run by clicking on test run result table.
```
Comment (hcgatewood) [inline docs/readmes/lte/hil_tests.md]: Go ahead and move this to the top of the doc, since it's probably the most important link

**Review by hcgatewood — APPROVED** [review]
Awesome! I was able to take a look at the dashboard. Thanks for adding this!

**Pull request merged**